spec/models/mixins/external_url_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe ExternalUrlMixin do
+  let(:test_class) do
+    Class.new(ActiveRecord::Base) do
+      def self.name; 'TestClass'; end
+      self.table_name = 'vms'
+      include ExternalUrlMixin
+    end
+  end
+
+  describe '#external_url=' do
+    before do
+      User.current_user = FactoryBot.create(:user)
+    end
+
+    let(:test_instance) do
+      i = test_class.create
+      i.external_url = 'https://www.hmpf.cz'
+      i
+    end
+
+    it 'sets url for the current user' do
+      expect(test_instance.external_urls.first.attributes).to include(
+        'resource_type' => 'TestClass',
+        'resource_id'   => test_instance.id,
+        'url'           => 'https://www.hmpf.cz',
```

Comment:
I think it would be better to use a URL meant for documentation and tests like `www.example.com`.

Reply:
ok